src/main/java/uk/gov/hmcts/reform/sscs/service/hmc/topic/HearingUpdateService.java

```diff
@@ -116,6 +117,15 @@ public void setHearingStatus(String hearingId, @Valid SscsCaseData sscsCaseData,
         hearing.getValue().setHearingStatus(hearingStatus);
     }
 
+    public void setDwpState(HmcStatus hmcStatus, @Valid SscsCaseData caseData) {
```

Comment:
Should this be public?

Reply:
Instead of making this return void it's clearer if this returns the state for the caller to set

Reply:
Yes this needs to be public for when it is called in ProcessHmcMessageService. Name has been updated to keep with naming convention of setters returning only void